scripts/userstyles.yml

```
@@ -410,6 +412,16 @@ userstyles:
     readme:
       app-link: https://google.com
     current-maintainers: []
+  google-contacts:
+    name: Google Contacts
+    categories: [social_networking]
+    # Review: there seems to be no matching icon in the
+    # simple icons library, adding one would be preferred
+    icon: google
```

Yeah it's a difficult icon to implement. We don't need to worry but if you are interested see https://github.com/simple-icons/simple-icons/issues/7226.
```suggestion
    icon: google
```